src/Mod/Path/Path/Op/Gui/Drilling.py

```diff
@@ -161,7 +172,8 @@ def getSignalsForUpdate(self, obj):
         signals.append(self.form.toolController.currentIndexChanged)
         signals.append(self.form.coolantController.currentIndexChanged)
         signals.append(self.form.ExtraOffset.currentIndexChanged)
-
+        signals.append(self.form.KeepToolDownEnabled.stateChanged)
+        
```

Comment:
Sorry , I'm not going waste time doing a third PR for white spaces.
If there are major changes required to merge , it may get looked into. 
If it gets ignored like that last one, at least I've got my diffs so I can stop FreeCAD crashing my tools. It would be nice if we could save other folks from tool crashes too.